pysilcam/__main__.py

```diff
@@ -698,13 +702,14 @@ def adminSTATS(logger, settings, overwriteSTATS, datafilename, datapath):
             # TODO: move this import to the top
             from pysilcam.fakepymba import silcam_name2time
             files = [f for f in sorted(os.listdir(datapath))
-                     if (f.endswith('.silc') or f.endswith('.bmp'))]
+                     if f.endswith('.silc' or f.endswith('.bmp'))]
```

Comment:
```suggestion
                     if f.endswith('.silc') or f.endswith('.bmp')]
```
I assume this is what you meant.